lib/talon.ex

```diff
@@ -77,6 +81,17 @@ defmodule Talon do
 
       def resource_names, do: @__resource_map__ |> Map.keys
 
+      def dashboard_names, do: [__MODULE__. dashboard_name]
```

Comment:
Yeah, I was cheating out of the gate and was still deciding on how dashboards and pages would ultimately relate (see above), so I went with just `dashboard_names` until I had a chance to revisit.